### PR TITLE
higher precision in lat long + support for manager to create icus

### PR DESCRIPTION
**Pull request by olivierteboul**

### CODE REVIEW

File: icubam/db/store.py [hunk 1]
```diff
@@ -322,9 +336,9 @@ def disable_icu(self, manager_user_id: int, icu_id: int):
     self.enable_icu(manager_user_id, icu_id, False)
 
   def assign_user_as_icu_manager(self, admin_user_id: int, manager_user_id: int,
-                                 icu_id: int):
+                                 icu_id: int, force: bool = False):
```
Comment (fpquintao) [inline icubam/db/store.py:339]: perhaps a little documentation on when it's fine to have force=true.

File: icubam/backoffice/templates/icu.html [hunk 2]
```diff
@@ -95,15 +95,15 @@ <h3 class="card-title">
...
+             <input type="number" step="0.0001" id="lat" name="lat"
                class="form-control"
                placeholder="Latitude"
                value="{%if icu.lat %}{{icu.lat}}{% end %}" required autofocus>
            </div>
 
            <div class="col-25">
              <label for="long">Longitude</label>
-             <input type="number" step="0.001" id="long" name="long"
+             <input type="number" step="0.0001" id="long" name="long"
```
Comment (fpquintao) [inline icubam/backoffice/templates/icu.html:106]: coooool

File: icubam/db/store.py (hunk 1 above)
Comment (olivierteboul) [reply icubam/db/store.py:339]: you are right. done

**Comment by olivierteboul** [comment]
Thanks Fred, added a comment

**Pull request merged**